src/app/core/configurable-enum/basic-autocomplete/basic-autocomplete.component.ts

```diff
@@ -230,48 +239,46 @@ export class BasicAutocompleteComponent<O, V = O>
   }
 
   onFocusIn() {
-    clearTimeout(this.delayedBlur);
-    if (!this.focused) {
-      if (this.multi) {
-        this.autocompleteForm.setValue("");
-      } else {
-        this.showAutocomplete();
-      }
-      this.focus();
+    if (this.multi) {
+      this.autocompleteForm.setValue("");
+    } else {
+      this.showAutocomplete();
```

Comment:
can we inline / simplify here? Why only showAutocomplete for single and for multi resetting the value is enough?